packages/ui/src/components/SideMenu/SideMenuCategory.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { Children } from "react";
+import { motion, useReducedMotion } from "framer-motion";
+
+import { useSideMenuContext } from "./SideMenuProvider";
+
+const commonStyles = {
+  "data-h2-background-color": `
+    base(transparent)
+    base:focus-visible(focus)
+
+    base:iap(secondary.light)
+    base:iap:focus-visible(focus)
+  `,
+  "data-h2-outline": "base(none)",
+  "data-h2-color": `
+    base(white)
+
+    base:selectors[.active](secondary.lighter)
+
+    base:all:focus-visible(black)
+    base:all:iap:focus-visible(black)
+  `,
+  "data-h2-width": "base(100%)",
+  "data-h2-text-align": "base(left)",
+  "data-h2-display": "base(block)",
+  "data-h2-font-weight": "base:selectors[.active](700)",
+};
+
+type SideMenuCategoryProps = {
+  title: string;
+  children: React.ReactNode;
+};
+
+const SideMenuCategory = ({ title, children }: SideMenuCategoryProps) => {
+  const ctx = useSideMenuContext();
+  const shouldReduceMotion = useReducedMotion();
+  const transitionConfig = { duration: shouldReduceMotion ? 0 : undefined };
+  const subitemCount = Children.toArray(children).length;
+  if (!subitemCount) return null; // hide empty categories
+  return (
+    <div
+      data-h2-display="base(flex)"
+      data-h2-flex-direction="base(column)"
+      data-h2-align-items="base(flex-start)"
+      data-h2-gap="base(x0.5)"
+      data-h2-align-self="base(stretch)"
+    >
+      <div
+        {...commonStyles}
+        data-h2-border-bottom="base(1px solid)"
+        data-h2-font-weight="base(700)"
+        data-h2-height="base(x1)"
+      >
+        <motion.span
+          transition={transitionConfig}
+          animate={ctx?.open ? { opacity: 1 } : { opacity: 0 }}
+        >
+          {!!ctx?.open && title}
+        </motion.span>
```

Comment:
Of course, the second I gave up and just submitted a review I found where the issue was coming from 🤦‍♀️ .

So, it is coming from the title popping in and out. It looks like height was added the parent to avoid this but it isn't 100% exact so it only reduces the effect. I'm not sure if this is the best way, but the following should resolve it. If you don't want the title to be announced to ATs you can just toggle the `aria-hidden` from `"true"` to `undefined` here instead.

```suggestion
        <motion.span
          data-h2-display="base(flex)"
          transition={transitionConfig}
          animate={ctx?.open ? { opacity: 1, width: "auto" } : { opacity: 0, width: 0 }}
        >
          <span>
            {title}
          </span>
        </motion.span>
```

Reply:
Good catch!  I found that this change had a weird wrap for titles with more than one word while the width was animating.
https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/45b6aeb3-3254-4b98-90b8-d88b81687fde
I added a "nowrap" to the span to avoid this.  Hopefully none of the French titles are long enough to require this wrap.  :crossed_fingers: 
[fix title pop](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8972/commits/1fa0035ffc1366d5de1087d443e9f2bbd98a52fa)

Reply:
oh yeah, that is a good call! LGTM 👍